src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoComponent } from '../TodoComponent';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  onDelete: (id: number) => void,
+  loadingTodoIds: number[],
+  onUpdateTodo: (id: number, todo: object) => void,
```

Comment:
Instead of using 'object' as a type, use the specific 'Todo' type to ensure better type safety 🌟